client/components/profile/ProfileEditor.jsx

```diff
@@ -23,16 +23,55 @@ ProfileEditor = class ProfileEditor extends Component {
       lastname: user.lastname || '',
       email: user.getEmail(),
       lookingForTeam: user.lookingForTeam || false,
+      inPersonAllowed: user.inPersonAllowed,
       bio: user.bio || '',
     };
   }
 
+  _gameTypeBox() {
+    let icon, header, content;
+    const footer = (
+      <span>For questions, see our <Link to="/contact">Contact Page</Link> or
+      our <Link to="/faq">FAQ</Link>.</span>
+    );
+
+    if (!this.state.inPersonAllowed) {
+      icon = "video camera";
+      header = "You are registered to play virtually";
+      content = (
+        <span>
+          Only WWU community members with in-person ticket codes are allowed to
+          participate in person for 2022. {footer}
+        </span>
+      );
+    } else {
+      icon = "group";
+      header = "You may play in person this year!";
+      content = (
+        <span>
+          You have purchased an in-person ticket code. If your team has the
```

Comment:
Just something I noticed, if we manually toggle their in-person play from the admin panel, they may not have "purchased an in-person ticket code."

Not a big deal though!

Reply:
Mm good point. Hadn't thought of that. Open to a wording change if you want to make it and push it up, otherwise I'm happy to leave as-is